pkg/packagekit/updates.jsx

```diff
@@ -1625,7 +1625,6 @@ class OsUpdates extends React.Component {
 }
 
 document.addEventListener("DOMContentLoaded", () => {
-    document.title = cockpit.gettext(document.title);
```

Comment:
That surprises me... I have never thought about this, but indeed other pages don't worry about this.